api/lds.proto

```diff
@@ -66,13 +66,27 @@ message FilterChainMatch {
 
   // If non-empty, an IP address and prefix length to match addresses when the
   // listener is bound to 0.0.0.0/::.
```

Comment:
Yes, I'm proposing that as an option. Is that simpler to understand? Not sure. Happy to leave it this way for now if we want to open an issue to think about it.